src/client/cl_main.c

```diff
@@ -3013,6 +3013,7 @@ void CL_Init(void)
 	Cvar_Get("cg_autoReload", "1", CVAR_ARCHIVE);
 	Cvar_Get("cg_weapaltReloads", "0", CVAR_ARCHIVE);
 	Cvar_Get("cg_weapaltSwitches", "1", CVAR_ARCHIVE);
+	Cvar_Get("cg_sharetimerText", "", CVAR_ARCHIVE);
```

Comment:
Very little reason to pollute newly added mod-specific cvars in the engine tbh.

Reply:
Thanks for checking. I just added the cvar in places where the cvar I used as example was.